client/src/Home.js

```diff
@@ -1,9 +1,169 @@
+import './Home.scss';
+import { Link } from 'react-router-dom';
+import uberLogo from './images/Uber-Open-Source Logo.png';
+import DMLogo from './images/DevMission Logo.jpg';
+import C4SFLogo from './images/Code4SF Logo.png';
+
 function Home() {
   return (
-    <main className="container">
-      <h1>Home</h1>
-      <p>Tech Equity (Test)</p>
-    </main>    
+    <main>
+      {/* Heading */}
+      <div className="top">
+        <div className="container">
+          <h1 className="h1"> <strong>Here's a catchy <br /> tidbit about <br /> how it works.</strong></h1>
+          {/*Figuring out how to make it scroll down automatically using the link tag*/}
+          <Link to="#roleSection" className="SecondaryColor btn btn-primary mt-5"> How you can help ↓</Link>
+        </div>
+      </div>
+      <div className="container">
```

Comment:
You forgot that, to properly use row/col classes, they should be put into a wrapping div with the container class, so that they get properly centered in the page.